### DISCUSSION TITLE
Which IDE does Daniel Kao use for developing and maintaining EinkBro?

### DISCUSSION THREAD

**Discussion by alchem1no** [plateaukao/einkbro#428] [General]
Which IDE do you use for developing and maintaining EinkBro? There is IntelliJ IDEA logo in ReadMe file which, however, does not lead to IntelliJ IDEA webpage but to the webpage related to Open Source support by JetBrains. And because of that I'm not sure which IDE is used for this project.

**Comment by plateaukao** [comment]
I mainly use Android Studio. and sometimes use IntelliJ.
